lib/simplecov.rb

```diff
@@ -67,25 +68,63 @@ def add_not_loaded_files(result)
         result = result.dup
         Dir[tracked_files].each do |file|
           absolute = File.expand_path(file)
-
-          result[absolute] ||= LinesClassifier.new.classify(File.foreach(absolute))
+          result[absolute] ||= RunFileCoverage.start(absolute)
         end
       end
 
       result
     end
 
+    #
+    # Unite the result so it wouldn't matter what coverage type was called
+    #
+    # @return [Hash]
+    #
+    def adapt_coverage_result
```

Comment:
Should methods `adapt_coverage_result`, `remove_useless_results`, `result_with_not_loaded_files` and `process_coverage_result` belong to public interface?

Reply:
Totally agree, they should be defined as private methods.